docs/xtradb-cluster-version-numbers.md

```diff
@@ -0,0 +1,34 @@
+# Understand version numbers
+
+A version number identifies the product release. The product contains the latest Generally Available (GA) features at the time of that release.
+
+<style>
+    table {
+        border-collapse: collapse;
+        width=100%;
+    }
+    table td {
+        border: 2px solid black;
+        padding: 8px;
+        text-align: center;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+</style>
+
+| 8.0.20| -11. | 2 |
+|---|---|---|
+| Base version | minor build | custom build |
+
+Percona uses semantic version numbering, which follows the pattern of base version, minor build, and an optional custom build. Percona assigns unique, non-negative integers in increasing order for each minor build release. The version number combines the base Percona Server for MySQL version number, the minor build version, and the custom build version, if needed.
+
+The version numbers for Percona XtraDB Cluster 8.0.20-11.2 define the following information:
+
+* Base version - the leftmost numbers indicate the Percona Server for MySQL version used as a base. An increase in base version resets the minor build version and the custom build version to 0. 
+
+* minor build version - an internal number that denotes which version that increases by one then the custom build fix is reset to 0.
```

Comment:
an internal number that denotes which version that increases - this sentence is incomplete

Reply:
fixed.